!!!SEIZE POWER HIJACK AMEND AUTOCRACY PR!!!

hongaar opened the pull request:
- Implemented `__all__` in github_api module (fixes PyCharms 'Cannot find reference 'xxx' in xxx.py', see https://stackoverflow.com/questions/23248017/cannot-find-reference-xxx-in-init-py-python-pycharm).

PlasmaPower commented on github_api/__init__.py:
```diff
@@ -25,7 +36,7 @@ class API(object):
     BASE_URL = "https://api.github.com"
     BASE_HEADERS = {
-        #"Accept": "application/vnd.github.v3+json"
+        # "Accept": "application/vnd.github.v3+json"
```
Already fixed on master.

hongaar committed:
Implemented __all__ in github_api module.
Fix PyCharms 'Cannot find reference 'xxx' in xxx.py', see https://stackoverflow.com/questions/23248017/cannot-find-reference-xxx-in-init-py-python-pycharm.

hongaar committed:
Ignore me please.

phil-r committed:
Merge branch 'master' into feature/meta
